tests/test_crawl.py

```diff
@@ -251,6 +251,19 @@ def start_requests(self):
     def test_open_spider_error_on_faulty_pipeline(self):
         settings = {
             "ITEM_PIPELINES": {
+                "tests.pipelines.LegacyZeroDivisionErrorPipeline": 300,
```

Comment:
If you change naming like suggestion above, don't forget to also change this one